packages/workspace/src/generators/new/new.ts

```diff
@@ -77,6 +69,46 @@ function generatePreset(host: Tree, opts: Schema) {
       }
     });
   });
+
+  function getPresetArgs(options: Schema) {
+    if (Object.values(Preset).some((val) => val === options.preset)) {
```

Comment:
Could use `Object.values(Preset).includes(options.preset)` for succinctness.

Reply:
The reason i did it this way is because you would get a ts lint error
`Argument of type 'string' is not assignable to parameter of type 'Preset'.`